README.md

```diff
@@ -1,11 +1,13 @@
 [![Build Status](https://travis-ci.org/Teamwork/httperr.svg?branch=master)](https://travis-ci.org/Teamwork/httperr) [![Codecov](https://img.shields.io/codecov/c/github/Teamwork/httperr.svg?style=flat)](https://codecov.io/gh/Teamwork/httperr) [![GoDoc](https://godoc.org/github.com/Teamwork/httperr?status.svg)](http://godoc.org/github.com/Teamwork/httperr)
+[![codecov](https://codecov.io/gh/Teamwork/httperr/branch/master/graph/badge.svg)](https://codecov.io/gh/Teamwork/httperr)
+[![GoDoc](https://godoc.org/github.com/Teamwork/httperr?status.svg)](https://godoc.org/github.com/Teamwork/httperr)
```

Comment:
Oh, because it was all on a single line I thought it was just the Travis one >_<